components/home/ActiveTripCard.tsx

```
@@ -0,0 +1,146 @@
+  const { defaultCurrency } = useDefaultCurrency();
+  const { tripsEnabled } = useTripsEnabled();
+  const router = useRouter();
+  const [spending, setSpending] = useState<{
+    totalSpent: number;
+    transactionCount: number;
+    currencyCode: string | null;
+  } | null>(null);
+
+  const activeTrip = activeTrips[0]; // Show the first/most recent active trip
+
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook specifies a dependency more specific that its captures: activeTrip?.id

This capture is more generic than...

...this dependency.


(lint/correctness/useExhaustiveDependencies)